nsv13/code/game/gamemodes/pvp/base_defense.dm

```diff
@@ -0,0 +1,225 @@
+
+/obj/machinery/nuclearbomb/selfdestruct/syndicate
+	name = "base self-destruct terminal"
+	desc = "A terminal which, when provided with an Osterhagen key, can trigger a chain reaction inside its planetary core, completely annihilating its surroundings and scrubbing all evidence of a Syndicate operation."
+	r_code = "73556"
+
+/atom/onSyndieBase()
+	. = ..()
+	var/area/AR = get_area(src)
+	if(istype(AR, /area/tartarus) || istype(AR, /area/hammurabi))
+		return TRUE
+
+/datum/game_mode/pvp/base_defense/OnNukeExplosion(off_station)
+	..()
+	nukes_left = 0
+	check_win()
+//Puce GANG
+/obj/item/disk/nuclear/syndicate_base
+	name = "Osterhagen key"
+	desc = "A disk which can initiate the Osterhagen protocol and scrub its assigned Syndicate base by triggering a series of nuclear warheads to be launched at the planet's core, causing complete annihilation. <b>It has the activation code stamped on it:</b><i>73556</i>"
+	icon = 'nsv13/icons/obj/items_and_weapons.dmi'
+	fake = FALSE //Yeah this disk is very, very real.
+
+
+//The inverse of stationloving. This forces the Syndicate PVP disk to not be on the station or overmap level.
+
+/datum/component/stationloving/stationhating/in_bounds()
+	if(SSticker.current_state <= GAME_STATE_PREGAME)
+		return TRUE
+	var/static/list/allowed_shuttles = typecacheof(list(/area/shuttle/syndicate, /area/tartarus, /area/hammurabi))
+	var/static/list/disallowed_centcom_areas = typecacheof(list(/area/abductor_ship, /area/awaymission/errorroom))
+	var/turf/T = get_turf(parent)
+	if (!T)
+		return FALSE
+	var/area/A = T.loc
+	if (SSmapping.level_trait(T.z, ZTRAIT_OVERMAP))
+		return FALSE
+	if (is_station_level(T.z))
+		return FALSE
+	if (is_centcom_level(T.z))
+		if (is_type_in_typecache(A, disallowed_centcom_areas))
+			return FALSE
+		return TRUE
+	if (is_reserved_level(T.z))
+		if (is_type_in_typecache(A, allowed_shuttles))
+			return TRUE
+
+	return FALSE
+
+//Override to plop the disk back to a syndie crew spawn rather than somewhere on the station.
+
+/datum/component/stationloving/stationhating/relocate()
+	var/targetturf = get_turf(pick(GLOB.syndi_crew_spawns)) //Fuck you for making me do this.
+	var/atom/movable/AM = parent
+	AM.forceMove(targetturf)
+	to_chat(get(parent, /mob), "<span class='danger'>You can't help but feel that you just lost something back there...</span>")
+	return targetturf
+
+/obj/item/disk/nuclear/syndicate_base/ComponentInitialize()
+	AddComponent(/datum/component/stationloving/stationhating)
+
+//Syndie PVP ship
+
+/area/tartarus
+	name = "Tartarus"
+	ambientsounds = list('nsv13/sound/ambience/leit_motif.ogg','nsv13/sound/ambience/wind.ogg','nsv13/sound/ambience/wind2.ogg','nsv13/sound/ambience/wind3.ogg','nsv13/sound/ambience/wind4.ogg','nsv13/sound/ambience/wind5.ogg','nsv13/sound/ambience/wind6.ogg')
+	noteleport = TRUE
+	icon_state = "syndie-ship"
+	has_gravity = TRUE
+
+/area/tartarus/underground
```

Comment:
Refactoring this to a sub-type of the mining area and causing it to handle cave generation would likely help with the lack of underground flavor.